app/client/routes/main.coffee

```diff
@@ -1,50 +1,74 @@
 BlazeLayout.setRoot('body')
 
+
 FlowRouter.route '/',
-  action: () ->
-    FlowRouter.go '/admin/surveys'
+  action: ->
+    if Meteor.user()
+      FlowRouter.go '/admin/surveys'
+    else
+      FlowRouter.go 'login'
+
+
+loggedIn = FlowRouter.group
+  triggersEnter: [ ->
+    unless Meteor.loggingIn() or Meteor.userId()
+      route = FlowRouter.current()
+      # unless route.route.name is 'login'
+        # Session.set 'redirectAfterLogin', route.path
+      FlowRouter.go 'login'
+  ]
+
+
+adminRoutes = loggedIn.group
+  prefix: '/admin'
```

Comment:
I wonder if we need to prefix the routes with 'admin'. Will we ever need routes associated with non-admin users?